admin_guide/topics/manage_crd_obj.adoc

```diff
@@ -0,0 +1,69 @@
+////
+manage custom objects created from a CRD
+
+Module included in the following assemblies:
+
+* admin_guide/custom_resource_definitions.adoc
+* dev_guide/creating_crd_objects.adoc
+////
+
+[id='manage-crd-obj-{context}']
+= Managing custom objects
+
+After you create objects, you can manage your custom resources.
+
+* To get information on a specific kind of custom resource, enter:
++
+----
+oc get <kind>
+----
++
+For example:
++
+----
+oc get crontab
+
+NAME                 KIND
+my-new-cron-object   CronTab.v1.stable.example.com
+----
++
+Note that resource names are not case-sensitive,
+and you can use either the singular or plural forms defined in the CRD,
+as well as any short name. For example:
++
+----
+oc get crontabs
+oc get crontab
+oc get ct
+----
+
+* You can also view the raw JSON data for a custom resource:
++
+----
+oc get <kind> -o yaml
+----
++
+You should see that it contains the custom <1> `cronSpec` and <2> `image` fields
```

Comment:
The <1> & <2> aren't rendering correctly in the build, though they do the job of calling out the line items.